livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/realtime_model.py

```diff
@@ -125,8 +123,6 @@ class RealtimeContent:
     """stream of text content"""
     audio_stream: AsyncIterable[rtc.AudioFrame]
     """stream of audio content"""
-    tool_calls: list[RealtimeToolCall]
-    """pending tool calls"""
```

Comment:
The reason why it's there is because RealtimeAPI is a 1/1 mapping to the OAI ws API